lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
 import Tab from './Tab';
+import propTypes from 'prop-types'
```

Comment:
When we import prop types, we usually will import it as "PropTypes". This way, we can tell that it is a global object that we can access properties/methods off of